tests/e2e/kcs/test_noobaa_db_backup_and_recovery.py

```diff
@@ -76,3 +81,57 @@ def test_noobaa_db_backup_and_recovery(
         # Verify everything running fine
         log.info("Verifying All resources are Running and matches expected result")
         self.sanity_helpers.health_check(tries=120)
+
+    @pytest.fixture()
+    def warps3(self, request):
+
+        warps3 = warp.Warp()
+        warps3.create_resource_warp(replicas=3, multi_client=True)
+
+        def teardown():
+            warps3.cleanup()
+
+        request.addfinalizer(teardown)
+        return warps3
+
+    def test_noobaa_db_backup_recovery_locally(
```

Comment:
polarion id, bugzilla marker are missing and you may have to skip it on unsupported platforms